zstash/globus.py

```diff
@@ -387,10 +402,24 @@ def globus_finalize(non_blocking: bool = False):
     global transfer_client
     global transfer_data
     global task_id
+    global gv_tarfiles_pushed
 
     last_task_id = None
 
     if transfer_data:
+        # DEBUG: review accumulated items in TransferData
```

Comment:
This is just a note explaining this code block, right? Not a TODO that still needs to be addressed?